src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -46,6 +46,26 @@ public partial class View
             return temp;
         });
 
+        /// <summary>
+        /// KeyInputFocusProperty
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty KeyInputFocusProperty = BindableProperty.Create("KeyInputFocus", typeof(bool), typeof(View), false, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var view = (View)bindable;
+            if (newValue != null)
+            {
+                Tizen.NUI.Object.SetProperty(view.swigCPtr, View.Property.KEY_INPUT_FOCUS, new Tizen.NUI.PropertyValue((bool)newValue));
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var view = (View)bindable;
+            string temp;
```

Comment:
It may be `bool`.

Reply:
Oh You're right.
I did the wrong type here. Fixed it.
Thank you for reviewing it!!